index.html

```diff
@@ -9,36 +9,238 @@
-        </div>
-        <nav>
-          <div class="menu-line"></div>
-          <div class="menu-line"></div>
-          <div class="menu-line"></div>
-        </nav>
-      </header>
-      <section>
-        <h1>Hey There.<br />I'm Anicet</h1>
-        <h2>I'am a Software Developer</h2>
-        <p>
-          I can help you build a product. featuer or <br />
-          website Look through some of my work <br />
-          and experience! if you see <br />
-          and have a project you need coded. <br />
-          don't hestiate to contact me.
-        </p>
-        <ul>
-          <li><a href=""><img src="images/Vector3.png" alt="" id="footer-img" /></a></li>
-          <li><a href=""><img src="images/linkedin.png" alt="" id="footer-img"/></a></li>
-          <li><a href=""><img src="images/Vector2.png" alt="" id="footer-img" /></a></li>
-          <li><a href=""><img src="images/Vector1.png" alt="" id="footer-img" /></a></li>
-          <li><a href=""> <img src="images/Vector.png" alt="" id="footer-img" /></a></li>
-        </ul>
-        <img src="images/illustration-header-mobile.png" class="side-img" alt=""/>
-      </section>
+    <div class="work-section">
+      <div class="wrapper">
+        <header>
+          <div class="right-side">
+            <a href="#"><div class="logo">Anicet Fantômas</div></a>
+          </div>
+          <nav>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+          </nav>
+        </header>
+        <section>
+          <h1>Hey There.<br />I'm Anicet</h1>
+          <h2>I'am a Software Developer</h2>
+          <p>
+            I can help you build a product. featuer or <br />
+            website Look through some of my work <br />
+            and experience! if you see <br />
+            and have a project you need coded. <br />
+            don't hestiate to contact me.
+          </p>
+          <ul>
+            <li>
+              <a href=""
+                ><img src="images/Vector3.png" alt="" id="footer-img"
+              /></a>
+            </li>
+            <li>
+              <a href=""
+                ><img src="images/linkedin.png" alt="" id="footer-img"
+              /></a>
+            </li>
+            <li>
+              <a href=""
+                ><img src="images/Vector2.png" alt="" id="footer-img"
+              /></a>
+            </li>
+            <li>
+              <a href=""
+                ><img src="images/Vector1.png" alt="" id="footer-img"
+              /></a>
+            </li>
+            <li>
+              <a href="">
+                <img src="images/Vector.png" alt="" id="footer-img"
+              /></a>
+            </li>
+          </ul>
+          <img
+            src="images/illustration-header-mobile.png"
+            class="side-img"
+            alt=""
+          />
+        </section>
+      </div>
+      <div class="container00 container1">
```

Comment:
As the previous reviewer requested, please, kindly use a `section` element instead of a `div` element, and give it a relevant class name of your choice.

```suggestion
      <section class="work-section">
```

This is as per [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). It is recommended to use HTML5 semantic tags because it will help to more accurately describe the relationship between HTML elements in your code.

Please, note that you would then need to change the closing tag accordingly.